indico/web/flask/session.py

```diff
@@ -74,14 +74,20 @@ def user(self):
                     flash(Markup(msg).format(merged_into_user.full_name), 'warning')
                 else:
                     flash(_(u'Your profile has been deleted.'), 'error')
+        if user and user.valid_login_num != self.get('_user_login_num'):
```

Comment:
what about `self.get('_user_login_num', 0)` to avoid logging everyone out who has an existing session where no login num had been set?